### ISSUE TITLE
'Text strings must be rendered within a <Text> component' when tap on KDO (KudosToken) from collectibles

### ISSUE THREAD

**Issue by churik** [status-im/status-react#10945]
# Bug Report

## Problem
Regression issue, not reproducible on 1.4.1
App crashes with error `Text strings must be rendered within a <Text> component` when tapping on KDO from Collectibles

#### Expected behavior

[comment]: # (Describe what you expected to happen. Please include designs if possible.)

#### Actual behavior

[comment]: # (Describe what actually happened. Please include screenshots if possible.)

#### Notes

For the developer who will be investigating. Is it related to another PR? Tips on where to start? 

#### Acceptance Criteria

Rules for the future PR to be accepted.

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)
**Prerequisites:**  multiaccount with KudosToken

- Open Status
- Restore multiaccount
- Open Status Wallet
- Tap on Collectibles
- Tap on KDO

### Additional Information

- Status version: nightly 13/07/2020
- Operating System: Android, iOS

#### Logs

[Status-debug-logs.zip](https://github.com/status-im/status-react/files/4912486/Status-debug-logs.zip)

**Comment by churik** [comment]
cc @hesterbruikman
